packages/cli/config-management/docs-configuration/src/convertDocsConfiguration.ts

```diff
@@ -19,20 +19,17 @@ export async function convertDocsConfiguration({
     absolutePathOfConfiguration: AbsoluteFilePath;
     context: TaskContext;
 }): Promise<DocsConfiguration> {
+    const { navigation, colors, favicon, logo, navbarLinks, title } = rawDocsConfiguration;
     return {
-        navigation: convertNavigationConfiguration(rawDocsConfiguration.navigation, absolutePathOfConfiguration),
-        title: rawDocsConfiguration.title,
+        navigation: convertNavigationConfiguration(navigation, absolutePathOfConfiguration),
+        title,
         logo:
```

Comment:
Looks like we're dropping the href? I think you'll need to edit the [`DocsConfiguration` type](https://github.com/fern-api/fern/blob/main/packages/cli/config-management/docs-configuration/src/DocsConfiguration.ts) and then update `publishDocs` to send the href to fdr